src/Bridge/Symfony/Resources/config/doctrine_mongodb_odm.xml

```diff
@@ -71,7 +71,7 @@
         <service id="fidry_alice_data_fixtures.persistence.persister.doctrine_mongodb.object_manager_persister"
                  class="Fidry\AliceDataFixtures\Bridge\Doctrine\Persister\ObjectManagerPersister"
                  lazy="true">
-            <argument type="service" id="doctrine_mongodb.odm.document_manager" />
+            <argument type="service" id="doctrine" />
```

Comment:
I suspect this would be a different one no? `doctrine` is for the ORM Registry isn't it?

Same comment for the two occurrences below

Reply:
Should be `doctrine_mongodb`, same for the purger and purger_factory